app/src/main/java/org/simple/clinic/activity/TheActivityController.kt

```diff
@@ -37,7 +45,14 @@ class TheActivityController @Inject constructor(
   private fun showAppLock(events: Observable<UiEvent>): Observable<UiChange> {
     val replayedCanShowAppLock = events
         .ofType<TheActivityLifecycle.Started>()
-        .filter { userSession.isUserLoggedIn() }
+        .flatMapSingle { _ ->
+          userSession.loggedInUser()
+              .firstElement()
+              .filter { it is Just<User> }
+              .map { (user) -> user!!.loggedInStatus }
+              .toSingle(NOT_LOGGED_IN)
```

Comment:
It took me good 2 minutes to understand that `NOT_LOGGED_IN` is the default value in `toSingle()`. Can this be improved for code readability?